src/openwhisk/static.js

```diff
@@ -129,13 +129,7 @@ function deliverPlain(owner, repo, ref, entry, root) {
         'X-Static': 'Raw/Static',
       },
     };
-  }).catch((rqerror) => {
-    console.error('REQUEST FAILED', rqerror.response.body.toString());
-    if (rqerror.statusCode === 404) {
-      return error(rqerror.response.body.toString(), 404);
-    }
-    return error(rqerror.message, rqerror.statusCode);
-  });
+  }).catch(rqerror => error(rqerror.response.body.toString(), rqerror.statusCode));
```

Comment:
Always responding with `body.toString()` makes sense, but I'd differentiate between the actual status codes we are seeing from `raw.githubusercontent.com`:
- 404: is actually a 404
- 400 (when there is no file name, just a directory): should be reported as a 404
- other cases: just pass it through